src/main/java/uk/gov/pay/publicauth/dao/AuthTokenDao.java

```diff
@@ -102,6 +102,13 @@ public Optional<LocalDateTime> revokeSingleToken(String accountId, TokenLink tok
                         .findFirst());
     }
 
+    public void revokeTokens(String accountId) {
+        jdbi.withHandle(handle ->
+                handle.createUpdate("UPDATE tokens SET revoked=(now() at time zone 'utc') WHERE account_id=:account_id")
```

Comment:
I think the `AND revoked IS NULL` like in the `revokeSingleToken` methods is still needed to prevent the revoked date from being updated if a token has already been revoked

Reply:
Yes, good shout. Thanks.